docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  solr:
+    image: solr:5.5.5
```

Comment:
Wanted to get it running on the closest version to what we were running it before, and start upgrading later.